### PR TITLE
[Reference cleanup]LPS-161154

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang @tinatian,

This PR is to convert method references to field references in document-library-service. Thanks!
https://issues.liferay.com/browse/LPS-161154

previous PR:https://github.com/liferay-core-infra/liferay-portal/pull/1430

bad filter https://github.com/liferay-core-infra/liferay-portal/pull/1430#discussion_r954260981 has been deleted in 7afe46e431c876d57565a76485d6f67109c9dd82

### CODE REVIEW

**Review by dantewang — APPROVED** [review]

**Comment by JanisZhang** [comment]
ci:test:document

**Comment by JanisZhang** [comment]
Hi @catarineaguiar, Could you please help me scan the test results? thanks!

**Comment by catarineaguiar** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
@JanisZhang no regressions found.

File: modules/apps/document-library/document-library-service/src/test/java/com/liferay/document/library/internal/search/SearchResultUtilDLFileEntryTest.java [hunk 1]
```diff
@@ -364,11 +365,16 @@ private SearchResultContributor _createSearchResultContributor() {
 		DLFileEntrySearchResultContributor dlFileEntrySearchResultContributor =
 			new DLFileEntrySearchResultContributor();
 
-		dlFileEntrySearchResultContributor.setClassNameLocalService(
+		ReflectionTestUtil.setFieldValue(
```
Comment (tinatian) [inline modules/apps/document-library/document-library-service/src/test/java/com/liferay/document/library/internal/search/SearchResultUtilDLFileEntryTest.java:368]: @JanisZhang Please remove the blank line here, see this comment for more details https://github.com/brianchandotcom/liferay-portal/pull/122468#issuecomment-1233491759

**Comment by JanisZhang** [comment]
Hi @tinatian, force-pushed. I only removed the blank line, based on https://github.com/brianchandotcom/liferay-portal/pull/122468#issuecomment-1233491759. Thank you!

**Comment by tinatian** [comment]
@shuyangzhou This pull is good to go.

Tina

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/122849

**Pull request closed without merging**